_assets/patches/geth/0037-new-cht.patch

```diff
@@ -1,31 +0,0 @@
-diff --git a/light/postprocess.go b/light/postprocess.go
```

Comment:
We don't need this patch anymore? It was added like today :D

Reply:
recent cht's were are a part the 1.8.14 release
i think this would be a common workflow to remove a cht patch after new geth release update